src/solc_0.8/polygon/child/asset/PolygonAssetERC721.sol

```diff
@@ -0,0 +1,27 @@
+//SPDX-License-Identifier: MIT
+// solhint-disable-next-line compiler-version
+pragma solidity 0.8.2;
+
+import {AssetBaseERC721} from "../../../assetERC721/AssetBaseERC721.sol";
+import {IChildToken} from "../../../common/interfaces/@maticnetwork/pos-portal/child/ChildToken/IChildToken.sol";
```

Comment:
use nodes_modules one

Reply:
We dont have `pos-portal` package in our repo to import the interface from there. Will refactor the file path.